src/OpenLoco/src/Ui/Windows/Construction/OverheadTab.cpp

```diff
@@ -413,10 +414,10 @@ namespace OpenLoco::Ui::Windows::Construction::Overhead
 
         self.activatedWidgets &= ~(1 << widx::checkbox_1 | 1 << widx::checkbox_2 | 1 << widx::checkbox_3 | 1 << widx::checkbox_4);
 
-        self.widgets[widx::checkbox_1].type = WidgetType::none;
-        self.widgets[widx::checkbox_2].type = WidgetType::none;
-        self.widgets[widx::checkbox_3].type = WidgetType::none;
-        self.widgets[widx::checkbox_4].type = WidgetType::none;
+        self.widgets[widx::checkbox_1].hidden = true;
+        self.widgets[widx::checkbox_2].hidden = true;
+        self.widgets[widx::checkbox_3].hidden = true;
+        self.widgets[widx::checkbox_4].hidden = true;
```

Comment:
I don't see these being assigned `false` anywhere in the diff. Is that a mistake?

Reply:
Yep it was missing, its done in the function setCheckbox, modified it.